sa/sa.go

```diff
@@ -1397,25 +1397,44 @@ func (ssa *SQLStorageAuthority) PauseIdentifiers(ctx context.Context, req *sapb.
 	return response, nil
 }
 
-// UnpauseAccount will unpause all paused identifiers for the provided account.
-// If no identifiers are currently paused, this is a no-op.
-func (ssa *SQLStorageAuthority) UnpauseAccount(ctx context.Context, req *sapb.RegistrationID) (*emptypb.Empty, error) {
+// UnpauseAccount uses up to 5 iterations of UPDATE queries each with a LIMIT of
+// 10,000 to unpause up to 50,000 identifiers and returns a count of identifiers
+// unpaused. If the returned count is 50,000 there may be more paused identifiers.
```

Comment:
I assume we'll be signalling to the Subscriber in the SFE that they'll want to run the unpause operation multiple times if we keep returning 50k?

Reply:
Yep!